tests/unit_tests/test_io_handler.py

```diff
@@ -8,68 +8,91 @@
 
 
 def test_get_output_directory(args_dict, io_handler):
-
+    # default adding label
     assert io_handler.get_output_directory(label="test-io-handler") == Path(
         f"{args_dict['output_path']}/output/simtools-output/test-io-handler/"
     )
 
-    assert io_handler.get_output_directory(label="test-io-handler", test=True) == Path(
+    # label and testing
+    assert io_handler.get_output_directory(label="test-io-handler", dir_type="test") == Path(
         f"{args_dict['output_path']}/output/test-output/test-io-handler/"
     )
 
-    assert io_handler.get_output_directory(label="test-io-handler", dir_type="model") == Path(
+    # label and subdirectory (no testing)
+    assert io_handler.get_output_directory(label="test-io-handler", sub_dir="model") == Path(
         f"{args_dict['output_path']}/output/simtools-output/test-io-handler/model"
     )
 
+    # label and subdirectory (no testing); simtools-results should have no effect
+    assert io_handler.get_output_directory(
+        label="test-io-handler", sub_dir="model", dir_type="simtools-results"
+    ) == Path(f"{args_dict['output_path']}/output/simtools-output/test-io-handler/model")
+
+    # label and subdirectory (testing)
     assert io_handler.get_output_directory(
-        label="test-io-handler", dir_type="model", test=True
+        label="test-io-handler", sub_dir="model", dir_type="test"
     ) == Path(f"{args_dict['output_path']}/output/test-output/test-io-handler/model")
 
+    # all following tests: plain_path tests
```

Comment:
Not entirely sure what you mean with that. Could you explain a bit more what warning you expect?

Reply:
I meant that it would be good if every time `io_handler.use_plain_output_path` is True and the user passes `label` that the user is informed about the argument passed in `label` not being taken into account. But your argument above that the io_handler is not going to be used directly by external users (only by developers) convinced me that we might indeed not need the warning.